### DISCUSSION TITLE
GitHub Copilot could not connect to server.Extension activation failed:"unable to verify the first certificate'

### DISCUSSION THREAD

**Discussion by JFXJL** [community/community#17610] [Copilot Conversations]
GitHub Copilot could not connect to server.Extension
activation failed:"unable to verify the first certificate'
How to solve this problem？-vscode

**Comment by DonRichards** [comment]
Just started getting this message as well. I finally "signed up" for the trial since it's been warning me that it's no longer in beta (I guess). Been using it since it was opened for public trials. I don't have an answer as to why just noting it's a problem with mine as well.

**Comment by DonRichards** [comment]
Found a solution that worked for me [here](https://github.com/orgs/github-community/discussions/19061#discussioncomment-2995893)

**Comment by renatodru** [comment]
The problem is about trust in chain certificate.
The solution is:
1 - install the extension in VSC "win-ca"
2 - change config "Win-ca: inject (Method to provide fetched root certificates to other externsions)" to **append**.
3 - restart VSC.

**Reply by zioalex** [reply]
it didn't work for me. Same error as before.
